frontend/src/container/QueryBuilder/QueryBuilder.tsx

```diff
@@ -2,7 +2,7 @@ import { PlusOutlined } from '@ant-design/icons';
 import { Button, Col, Row } from 'antd';
 import { MAX_FORMULAS, MAX_QUERIES } from 'constants/queryBuilder';
 // ** Hooks
-import { useQueryBuilder } from 'hooks/useQueryBuilder';
+import { useQueryBuilderContext } from 'hooks/queryBuilder/useQueryBuilderContext';
```

Comment:
can we use `useQueryBuilder` while using `useQueryBuilderContext` though we are grabbing the value from React.Context only ...